### PR TITLE
Corrects the Typo on SRS guide - Closes 1587

### LINKED ISSUE #1587: Correct typo in the `Social Recovery System` tutorial
### Problem

- On the [Social Recovery System tutorial](https://lisk.com/documentation/tutorial/srs.html), the first section: **1. Project Setup** has a typo in it. 

Screenshot 2022-07-28 at 11 18 40 AM

### Solution
- The word **ptoject** should be corrected to **project**.

Affected branch: `main/development`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #[1587](https://github.com/LiskHQ/lisk-docs/issues/1587)

### CODE REVIEW

File: docs/modules/ROOT/pages/tutorial/srs.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ In addition to the blockchain application, a **frontend application** will also
 
 Create a new folder which will contain all the files for the SRS app.
 The Social Recovery System consists of two applications: The blockchain application and the frontend application.
-First, create the folder for the blockchain application, initialize a new Node.js ptoject and install the xref:{url_lisk_sdk}[Lisk SDK] as dependency.
+First, create the folder for the blockchain application, initialize a new Node.js project and install the xref:{url_lisk_sdk}[Lisk SDK] as dependency.
```
Comment (chris529) [inline docs/modules/ROOT/pages/tutorial/srs.adoc]:
```suggestion
First, create the folder for the blockchain application, initialize a new Node.js project and install the xref:{url_lisk_sdk}[Lisk SDK] as a dependency.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Looks good, just one very minor point 👍

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by Tschakki — APPROVED** [review]
Thanks!

**Pull request merged**